Azure-Samples/communication-services-android-calling-hero
# Feature of static participant list

## Purpose

* Enable viewing a static Participant List

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## Test the code
* Ensure participant list correctly represents the current list of participants in the call
* Ensure a full list of participants only takes up half the screen and is scrollable

## Other Information
